src/views/Layout.jsx

```diff
@@ -10,17 +15,99 @@ import './Layout.css';
  * defined in `App.jsx`.
  */
 
-export function Layout() {
+export function Layout({ data, listToken }) {
+	const location = useLocation();
+	const currentPath = location.pathname;
+
+	const [showListToken, setShowListToken] = useState(false);
+	const [copiedToken, setCopiedToken] = useState(false);
+
+	function toggleListToken() {
+		setShowListToken((prevState) => !prevState);
+	}
+
+	function copyTokenToClipboard() {
+		window.navigator.clipboard.writeText(listToken);
+		setCopiedToken(true);
+
+		setTimeout(() => {
+			setShowListToken(false);
+			setCopiedToken(false);
+		}, 2000);
+	}
+
 	return (
 		<>
-			<div className="Layout">
-				<header className="Layout-header">
-					<h1>Smart shopping list</h1>
-				</header>
+			<div className="h-screen md:max-w-md md:mx-auto">
+				{listToken && (
+					<header className="h-auto my-8">
+						<div className="flex items-center gap-4">
+							<img src="/img/logo.svg" alt="Shroomy logo" className="w-10" />
+							<h1 className="font-logo text-3xl mr-auto">Shroomy</h1>
+							{currentPath !== '/about' && (
+								<button
+									className="tooltip flex justify-center"
+									onClick={toggleListToken}
+								>
+									<FontAwesomeIcon
+										icon={faArrowUpFromBracket}
+										className="h-5 hover:text-main"
+									></FontAwesomeIcon>
+									<span className="tooltiptext icon mt-1">
+										Share with others
+									</span>
+								</button>
```

Comment:
Testing out the deploy preview with VoiceOver, this button gets announced as just "button", which doesn't tell the user what clicking this button will do.

To fix that, you can use the [`aria-describedby`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-describedby) attribute to label the button using the text inside the span. (You'll also need to add an `id` to the span.)

That way, the button will be announced as "Share with others, button"

```suggestion
								<button
									className="tooltip flex justify-center"
									onClick={toggleListToken}
									aria-describedBy="share-button-tooltip"
								>
									<FontAwesomeIcon
										icon={faArrowUpFromBracket}
										className="h-5 hover:text-main"
									></FontAwesomeIcon>
									<span className="tooltiptext icon mt-1" id="share-button-tooltip">
										Share with others
									</span>
								</button>
```